crates/sui-core/src/transaction_driver/unit_tests/effects_certifier_tests.rs

```diff
@@ -1148,10 +1154,12 @@ async fn test_aborted_with_multiple_effects() {
             0 => WaitForEffectsResponse::Executed {
                 effects_digest: effects_digest_1, // from fastpath
                 details: None,
+                fast_path: false,
             },
             1 => WaitForEffectsResponse::Executed {
                 effects_digest: effects_digest_2, // from fastpath
                 details: None,
+                fast_path: false,
```

Comment:
I wonder if it is possible to use `..Default::default()` for this variant of enum.

Reply:
Yes this sounds reasonable